mriqc/workflows/pet/base.py

```diff
@@ -0,0 +1,349 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+"""
+Functional workflow
+===================
+
+.. image :: _static/functional_workflow_source.svg
+
+The functional workflow follows the following steps:
+
+#. Sanitize (revise data types and xforms) input data, read
+   associated metadata and discard non-steady state frames.
+#. :abbr:`HMC (head-motion correction)` based on ``3dvolreg`` from
+   AFNI -- :py:func:`hmc`.
+#. Skull-stripping of the time-series (AFNI) --
+   :py:func:`fmri_bmsk_workflow`.
+#. Calculate mean time-series, and :abbr:`tSNR (temporal SNR)`.
+#. Spatial Normalization to MNI (ANTs) -- :py:func:`epi_mni_align`
+#. Extraction of IQMs -- :py:func:`compute_iqms`.
+#. Individual-reports generation --
+   :py:func:`~mriqc.workflows.functional.output.init_pet_report_wf`.
+
+This workflow is orchestrated by :py:func:`fmri_qc_workflow`.
+"""
+
+from nipype.interfaces import utility as niu
+from nipype.pipeline import engine as pe
+
+from mriqc import config
+from mriqc.workflows.pet.output import init_pet_report_wf
+
+
+def pet_qc_workflow(name='petMRIQC'):
```

Comment:
since this is all new, perfect opportunity to start off using the nipreps coding style
```suggestion
def init_pet_qc_wf(name='pet_qc_wf'):
```